process_frames.py

```diff
@@ -279,6 +285,71 @@ def run_ocr(self, image):
         else:
             print("No entry found")        
         return res, highlighted_image, annotations
+    
+    def call_openai_translation_api(self, content, target_lang):
```

Comment:
this should probably be broken to a separate file and combined with the gpt vision api calls

Reply:
Done